activerecord/test/cases/coders/yaml_column_test.rb

```diff
@@ -14,15 +14,15 @@ def test_type_mismatch_on_different_classes_on_dump
         error = assert_raises(SerializationTypeMismatch) do
           coder.dump("a")
         end
-        assert_equal %{Attribute `attr_name` was supposed to be a Array, but was a String. -- "a"}, error.to_s
+        assert_equal %{can't dump `attr_name`: input was supposed to be a Array, but was a String. -- "a"}, error.to_s
       end
 
       def test_type_mismatch_on_different_classes
         coder = YAMLColumn.new("attr_name", Array)
         error = assert_raises(SerializationTypeMismatch) do
           coder.load "--- foo"
         end
-        assert_equal %{Attribute `attr_name` was supposed to be a Array, but was a String. -- "foo"}, error.to_s
+        assert_equal %{can't load `attr_name`: input was supposed to be a Array, but was a String. -- "foo"}, error.to_s
```

Comment:
Can we use a different name other than `attr_name` in the tests? My paranoia tells me we could accidentally have missed the string interpolation and would literally print out `attr_name` regardless of the attribute's name :)